app/assets/javascripts/statuses.js

```diff
@@ -0,0 +1,77 @@
+$(function() {
+  $(".proxy-version").each(function(index, item) {
+    var item = $(item);
+    var url = item.data('url');
+    $.ajax({
+      type: 'get',
+      url:  url,
+      success: function(response) {
+        setItemMessage(item, response);
+      }
```

Comment:
is there a value in handing error? what happens if there is no response? (ditto for the other ajax below)